src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/FilterMutectCalls.java

```diff
@@ -11,23 +11,28 @@
 import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
 import org.broadinstitute.barclay.help.DocumentedFeature;
 import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
-import org.broadinstitute.hellbender.cmdline.programgroups.VariantProgramGroup;
-import org.broadinstitute.hellbender.engine.*;
+import org.broadinstitute.hellbender.cmdline.programgroups.VariantFilteringProgramGroup;
+import org.broadinstitute.hellbender.engine.FeatureContext;
+import org.broadinstitute.hellbender.engine.ReadsContext;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.engine.VariantWalker;
+import org.broadinstitute.hellbender.tools.exome.FilterByOrientationBias;
+import org.broadinstitute.hellbender.tools.walkers.contamination.CalculateContamination;
 import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
 import org.broadinstitute.hellbender.utils.variant.GATKVCFHeaderLines;
-import org.broadinstitute.hellbender.tools.exome.FilterByOrientationBias;
 
 import java.io.File;
-import java.util.*;
+import java.util.Set;
 import java.util.stream.Collectors;
 
 /**
  * Filter SNVs and indels from a Mutect2 callset.
  *
  * <p>
- *     FilterMutectCalls encapsulates GATK3 MuTect2's filtering functionality.
+ *     FilterMutectCalls encapsulates GATK3 MuTect2's filtering functionality and adds additional filters.
  *     GATK4 Mutect2 retains variant calling and some prefiltering.
- *     Separating calling and filtering functionalities into two tools better enables an iterative filtering process
+ *     Thresholds for filters are contained in {@link M2FiltersArgumentCollection} and described in <a href='https://github.com/broadinstitute/gatk/tree/master/docs/mutect/mutect.pdf' target='_blank'>.
```

Comment:
- Close out hyperlink with `</a>`, so `<a href='https://github.com/broadinstitute/gatk/tree/master/docs/mutect/mutect.pdf' target='_blank'>https://github.com/broadinstitute/gatk/tree/master/docs/mutect/mutect.pdf</a>`. Otherwise have entire doc portion hyperlinked:

![screenshot 2017-12-11 15 53 14](https://user-images.githubusercontent.com/11543866/33853290-6cd2eb54-de8b-11e7-8bcc-aec5da14a3d7.png)

Reply:
done